### DISCUSSION TITLE
Comenzar

### DISCUSSION THREAD

**Discussion by josemiguelfz** [Launch-X-Latam/MisionFrontEnd#23] [Q&A]
Tenemos que iniciar por nuestra cuenta o nos esperamos al próximo live?

**Comment by JosafatJimenezB** [comment]
A cuenta de que? si es de github pues ya la tienes, solo únete a la organización para no perderte nada y para agregar valor a tu cuenta, saludos!

**Comment by Emilio17MC** [comment]
Es recomendable que empieces a leer el contenido ya publicado en el repositorio, para que en el siguiente live complementes con la explicación del Mission Commander.

**Reply by OATS74** [reply]
buenos dias el live que hicieron donde se ubica que me lo perdi

**Reply by JosafatJimenezB** [reply]
aqui #22

**Comment by doguedogue** [comment]
Es buena idea que avances por tu cuenta para aprovechar al máximo las sesiones y comentar tus dudas. 🚀
Practicar practicar y practicar, si es html, escribelo, pruebalo, mira que falla, aqui la curiosidad no mata al gato 😀

https://github.com/Launch-X-Latam/MisionFrontEnd
